scripts/ci/verify_linter.py

```diff
@@ -0,0 +1,208 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+"""
+This script is used to verify linter on extensions.
+
+It's only working on ADO by default. If want to run locally,
+please update the target branch/commit to find diff in function find_modified_files_against_master_branch()
+"""
+
+import os
+import json
+from subprocess import check_output, check_call
+from pkg_resources import parse_version
+
+
+def separator_line():
+    print('-' * 100)
+
+
+class ModifiedFilesNotAllowedError(Exception):
+    """
+    Exception raise for the scenario that modified files is conflict against publish requirement.
+    Scenario 1: if modified files contain only src/index.json, don't raise
+    Scenario 2: if modified files contain not only extension code but also src/index.json, raise.
+    Scenario 3: if modified files don't contain src/index.json, don't raise.
+    """
+
+    def __str__(self):
```

Comment:
`__repr_`? `__str__` is also ok.

Reply:
Yeah. __repr__ is for debugging and developing, __str__ is for user and readibility.